src/EPR.Calculator.API/Controllers/CalculatorController.cs

```diff
@@ -172,11 +173,76 @@ public IActionResult GetCalculatorRuns([FromBody] CalculatorRunsParamsDto reques
         }
 
         [HttpGet]
-        [Route("calculatorRuns/{id}")]
-        public IActionResult GetCalculatorRun()
+        [Route("calculatorRuns/{runId}")]
+        public IActionResult GetCalculatorRun(int runId)
```

Comment:
Shall we surround with a try catch and return 500 Internal server error if any other exceptions?

Reply:
Yeah agree, Done the changes now. Thanks